depth_first_traversal/DepthFirstTraversal.java

```diff
@@ -0,0 +1,38 @@
+class DepthFirstTraversal {
+
+    public static int[] arr;    // DFS traversal path
+    public static int k = 0;
```

Comment:
@jsroyal I already said my codacy is not running on this repo right now. It's @sandeep201451066 . 
@sandeep201451066 : Can you please delete algos from your codacy account.